README.md

```diff
@@ -251,6 +251,13 @@ The Instana [AutoTrace WebHook](https://www.ibm.com/docs/en/obi/current?topic=ku
 
 ## Release History
 
+### 1.11.0 (2025-01-21)
+
+  * Add support for NGINX 1.27.3
+  * Add support for opentracing 0.38.0
```

Comment:
The NGINX OpenTracing version is completely internal. It is mentioned at https://www.ibm.com/docs/en/instana-observability/current?topic=technologies-monitoring-nginx#distributed-tracing-for-nginx-nginx-plus-and-openresty 
and we should even drop it there by putting `latest` inside. We should only mention it if there are real code changes coming in from there.
```suggestion
```